utils/utils.h

```diff
@@ -124,9 +124,15 @@
 
 # define  MSG(fd, fmt, ...)                                          \
           do {                                                       \
+            int len;                                                 \
+            char buf[8192] = {0};                                    \
```

Comment:
What if there are 10000 block volumes and we run '# gluster-block list <BHV> --json-pretty' :-)
We cannot predict this buffer size @lxbsz .

Reply:
Yeah, how about malloc a larger one dynamically ?

Reply:
Allocating memory for this per command basis seems unwanted for me, I'm sure you must be thinking the same :-)
Let me think on this tomorrow and get back to you.

Thanks!

Reply:
Yeah, sure.
This was also why I wanted to add one MSG_OUT instead.

Reply:
1. How about using GB_ASPRINTF ? (which involves copy)
2. or we may have to use va_start(), va_end() to parse __VA_ARGS__ one by one our self (this may be complicating)
3. or lets not complicate this, lets use MSG_OUT (Why have one more macro for achieving same thing ?)
4. or just follow what was there before (this is generic and lets caller decide what to do)

I will leave it to you :-)

Thanks!

Reply:
> 1. How about using GB_ASPRINTF ? (which involves copy)
This seems work well and looks good to me.

> 2. or we may have to use va_start(), va_end() to parse **VA_ARGS** one by one our self (this may be complicating)
> 3. or lets not complicate this, lets use MSG_OUT (Why have one more macro for achieving same thing ?)
> 4. or just follow what was there before (this is generic and lets caller decide what to do)
> 
For this IMO we'd better make all the LOG/MSG the same, no need to add the extra '\n' for all of them normally, expect user want to add more than one '\n' at the end.
Or it will be confusing why some have and other do not.

Thanks.

> I will leave it to you :-)
> 
> Thanks!